.drone.yml

```diff
@@ -56,11 +56,11 @@ steps:
 
   - name: pre-build
     pull: always
-    image: webhippie/nodejs:latest
+    image: mhart/alpine-node:10 # this is kept at the lowest version of node that we support
```

Comment:
Is there a particular reason we are using this instead of `node:10-alpine3.9`?

Reply:
The `node` images are huge because of C++ dependencies for native modules which we (currently) don't need. This one here is pretty minimal while still having version tags (which are missing from `webhippie/nodejs`).

Reply:
I'll switch to `node:12-slim`